github_connector/models/github.py

```diff
@@ -74,9 +74,10 @@
 
 class Github(object):
 
-    def __init__(self, github_type, login, password, max_try):
+    def __init__(self, github_type, max_try, login="", password="", token=""):
```

Comment:
I'm not sure that changing the order of the args of a function in a deployed version is a good idea. Is it possible just to add a last args token="" at the end ?

Reply:
I restored the order